Gruntfile.js

```diff
@@ -703,7 +712,7 @@ module.exports = function (grunt) {
       'autoprefixer',
       'connect:test',
       'add-redirect-uri',
-      'protractor:default',
+      (grunt.option('mac') ? 'protractor:mac' : 'protractor:default'),
```

Comment:
you need to do the same thing on line 708 above where you see protractor:default